src/elements/element.html

```diff
@@ -113,16 +118,16 @@
             superCtor.finalize();
           }
           this.finalized = true;
-          if (this.hasOwnProperty('is') && this.is) {
+          if (this.hasOwnProperty(goog.reflect.objectProperty('is', this)) && this.is) {
             Polymer.telemetry.register(proto);
           }
           let config = this._ownConfig;
           if (config) {
             this._finalizeConfig(config);
           }
-          if (this.template) {
-            let template = this.template.cloneNode(true);
-            this._finalizeTemplate(template);
+          let template = this.template;
```

Comment:
remove this symbol and do `this.template.cloneNode(true)` where `_finalizeTemplate` is called?